servicebus/data-plane/azure-servicebus/src/test/java/com/microsoft/azure/servicebus/ConfigValidateTestBase.java

```diff
@@ -0,0 +1,21 @@
+package com.microsoft.azure.servicebus;
+
+import org.junit.Assume;
+import org.junit.BeforeClass;
+
+public class ConfigValidateTestBase {
```

Comment:
I think calling this TestBase is sufficient.

Reply:
updated